bridge/src/main/java/com/livefront/bridge/BridgeDelegate.java

```diff
@@ -43,9 +43,21 @@
         mSavedStateHandler = savedStateHandler;
     }
 
+    void clear(@NonNull Object target) {
+        String uuid = mObjectUuidMap.get(target);
+        if (uuid == null) {
+            return;
+        }
+        clearDataFromDisk(uuid);
```

Comment:
Okay, whatever you do will be fine then.  Thanks for clarifying that they're weak references.